sensor/common/image/service_impl_test.go

```diff
@@ -0,0 +1,280 @@
+package image
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/expiringcache/mocks"
+	"github.com/stackrox/rox/sensor/common"
+	mocks2 "github.com/stackrox/rox/sensor/common/image/mocks"
```

Comment:
Maybe we could name those a bit clearer?

```suggestion
	cacheMocks "github.com/stackrox/rox/pkg/expiringcache/mocks"
	"github.com/stackrox/rox/sensor/common"
	imageMocks "github.com/stackrox/rox/sensor/common/image/mocks"
```

Reply:
Done